public/scss/components/Footer.module.scss

```
@@ -2,89 +2,99 @@
 
 .footer
 {
-  @include fixed(auto,auto,24px,auto);
-  padding: 0 18px;
-  max-height: 58px;
-  height: 100%;
-  width: 100%;
-  z-index: 10;
+  @include flex(column, center, flex-start);
+  background: $color_rose_bg2;
+  padding: 24px 0 0;
 
-  &_nav
+  &Navigasi
   {
-    @include flex(row, center, space-evenly);
-    background: $color_white;
-    border-radius: 56px;
-    box-shadow: 0px -4px 20px rgba(0, 0, 0, 0.05);
-    width: 100%;
-    max-width: 340px;
+    @include fixed(auto,auto,24px,auto);
```

```suggestion
    @include fixed(auto, auto, 24px, auto);
```
Boleh dibuat seperti di atas mba agar lebih seragam mungkin?